src/vai_lab/Environment/Environment_core.py

```diff
@@ -11,31 +11,29 @@ def set_avail_plugins(self, avail_plugins: PluginSpecsInterface) -> None:
     def set_data_in(self, data_in: DataInterface) -> None:
         self._data_in = data_in
 
-    def _load_plugin(self, plugin_name: str) -> None:
+    def _load_plugin(self, data_in) -> None:
+        self._plugin_name = self._module_config["plugin"]["plugin_name"]
```

Comment:
Is this `self._plugin_name` variable assigned here used often? Is it a convenience variable? It looks like the same information can always be accessed directly from `self._module_config["plugin"]["plugin_name"]?

Reply:
It was thought to be a convenience variable, but you are right we never used it. I just removed it.